### PR TITLE
Updates for completeness of gas initialization and plumerise

### PR DESCRIPTION
**Pull request by meganbela**
TYPE: enhancement

KEYWORDS: chemistry initialization, plumerise

SOURCE: internal

DESCRIPTION OF CHANGES:  
1. Added CH4 and CO2 initialization to RACM_SOA_VBS_KPP (chem_opt=108) and RACM_SOA_VBS_KPP_AQCHEM (chem_opt=109).
2. Added NH3 and CH4 to plumerise transport for biomass_burn_opt=1.
3. Updated CO2 to 400 ppm.

LIST OF MODIFIED FILES: 	
chem/chemics_init.F
chem/module_plumerise1.F

TESTS CONDUCTED: none

### CODE REVIEW

File: chem/module_plumerise1.F [hunk 1]
```diff
@@ -110,6 +110,8 @@ subroutine plumerise_driver (id,ktau,dtstep,                           &
                ebu(i,kts,j,p_ebu_no2)=ebu_in(i,1,j,p_ebu_in_no2)
                ebu(i,kts,j,p_ebu_co)=ebu_in(i,1,j,p_ebu_in_co)
                ebu(i,kts,j,p_ebu_co2)=ebu_in(i,1,j,p_ebu_in_co2)
+              ebu(i,kts,j,p_ebu_nh3) = ebu_in(i,1,j,p_ebu_in_nh3)
```
Comment (davegill) [inline chem/module_plumerise1.F]: For consistency, let's follow the indenting pattern, and also the "no space around the equal signs".

File: chem/chemics_init.F [hunk 2]
```diff
@@ -677,13 +677,13 @@ subroutine chem_init (id,chem,emis_ant,scalar,dt,bioemdt,photdt,chemdt,stepbioe,
 !!! TUCCELLA
    if( .NOT. config_flags%restart ) then
    kpp_select: SELECT CASE(config_flags%chem_opt)
-     CASE (GOCARTRACM_KPP,RACM_KPP,RACMPM_KPP,RACMSORG_KPP,RACM_MIM_KPP,RACM_ESRLSORG_KPP, &
-           RACMSORG_AQCHEM_KPP,RACM_ESRLSORG_AQCHEM_KPP)
+CASE(GOCARTRACM_KPP,RACM_KPP,RACMPM_KPP,RACMSORG_KPP,RACM_MIM_KPP,RACM_ESRLSORG_KPP,&
```
Comment (davegill) [inline chem/chemics_init.F]: No reason to change this line, right?

**Comment by davegill** [comment]
Is there any way to say, even if not shown, that the CH4 and CO2 are indeed initialized when using chem_opt 108, 109? Similar for NH3 and CH4 for plumerise?

**Comment by davegill** [comment]
@meganbela 
We have requested small changes. Permissions do not allow us to modify the proposed change.

File: chem/module_plumerise1.F (hunk 1 above)
Comment (meganbela) [reply chem/module_plumerise1.F]: Done

File: chem/chemics_init.F (hunk 2 above)
Comment (meganbela) [reply chem/chemics_init.F]: Yes, we are adding RACM_SOA_VBS_KPP and RACM_SOA_VBS_KPP_AQCHEM to the initialization of CO2, CH4, etc.

**Comment by meganbela** [comment]
> Is there any way to say, even if not shown, that the CH4 and CO2 are indeed initialized when using chem_opt 108, 109? Similar for NH3 and CH4 for plumerise?

Say where?

File: chem/chemics_init.F (hunk 2 above)
Comment (davegill) [reply chem/chemics_init.F]: Line 680 can be identical, there is no need to change it, the original indenting was correct

File: chem/module_plumerise1.F (hunk 1 above)
Comment (davegill) [reply chem/module_plumerise1.F]: line 113 needs to be indented, for consistency no spaces around the "=" sign, make it look the the original code; similar comments for line 115

**Comment by davegill** [comment]
@ravanah @kayeekayee 
Ravan and Ka Yee,
Would you please take a look at this PR. We are closing the repo to v4.1 very soon. This is quite small.

File: chem/chemics_init.F (hunk 2 above)
Comment (meganbela) [reply chem/chemics_init.F]: Ok

**Comment by kayeekayee** [comment]
Approved.

**Pull request merged**